Tests/Integration/MSFT_xIISServerLevelIntegration.Tests.ps1

```diff
@@ -1,53 +1,90 @@
 ﻿######################################################################################
 # Integration Tests for DSC Resource for IIS Server level defaults
-# 
+#
 # These tests change the IIS server level configuration but roll back the changes at the end
 # so they should be save to run.
-# Run as an elevated administrator 
+# Run as an elevated administrator
 # At this time, we don't have tests for all changable properties, but it should be easy to add more tests.
 ######################################################################################
 
-$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+# should check for the server OS
+if($env:APPVEYOR_BUILD_VERSION)
+{
+     Add-WindowsFeature Web-Server -Verbose
+}
+
+$DSCModuleName  = 'xWebAdministration'
+
+$moduleRoot = "${env:ProgramFiles}\WindowsPowerShell\Modules\$DSCModuleName"
+
+if(-not (Test-Path -Path $moduleRoot))
+{
+    $null = New-Item -Path $moduleRoot -ItemType Directory
+}
+else
+{
+    # Copy the existing folder out to the temp directory to hold until the end of the run
+    # Delete the folder to remove the old files.
+    $tempLocation = Join-Path -Path $env:Temp -ChildPath $DSCModuleName
+    Copy-Item -Path $moduleRoot -Destination $tempLocation -Recurse -Force
+    Remove-Item -Path $moduleRoot -Recurse -Force
+    $null = New-Item -Path $moduleRoot -ItemType Directory
+}
+
+Copy-Item -Path $PSScriptRoot\..\..\* -Destination $moduleRoot -Recurse -Force -Exclude '.git'
+
+if (Get-Module -Name $DSCModuleName -All)
+{
+    Get-Module -Name $DSCModuleName -All | Remove-Module
+}
+
+Import-Module -Name $(Get-Item -Path (Join-Path $moduleRoot -ChildPath "$DSCModuleName.psd1")) -Force
+
+if (($env:PSModulePath).Split(';') -ccontains $pwd.Path)
+{
+    $script:tempPath = $env:PSModulePath
+    $env:PSModulePath = ($env:PSModulePath -split ';' | Where-Object {$_ -ne $pwd.path}) -join ';'
+}
 
```

Comment:
We should add the Script Execution Policy code here:

# Preserve and set the execution policy so that the DSC MOF can be created
$executionPolicy = Get-ExecutionPolicy
if ($executionPolicy -ne 'Unrestricted')
{
    Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Force
    $rollbackExecution = $true
}